src/team_lib/mhba/xccl_mhba_team.c

```diff
@@ -495,10 +492,31 @@ xccl_status_t xccl_mhba_team_create_post(xccl_tl_context_t  *context,
             goto wc_alloc_fail;
         }
         memset(mhba_team->cq_completions, 0, sizeof(mhba_team->cq_completions));
+        /* allocate buffer for noninline UMR registration, has to be 2KB aligned */
+        umr_buf_size = align(sizeof(struct mlx5_wqe_umr_repeat_ent_seg) *
+                                     (mhba_team->node.sbgp->group_size + 1),
+                             64);
+        if (posix_memalign(&mhba_team->node.umr_entries_buf, 2048,
+                           umr_buf_size) != 0) {
+            xccl_mhba_error(
+                    "failed to allocate %zu bytes for noninline UMR buffer: %m",
+                    sizeof(struct mlx5_wqe_umr_pointer_seg) *
```

Comment:
you have 2 format specifiers and only 1 value provided. additionally you can use umr_buf_size as argument